### PR TITLE
Add keyboard selected to Autocomplete

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by szok** [comment]
autocomplete

**Comment by szok** [comment]
autocomplete

File: src/ralph/static/src/js/ralph-autocomplete.js [hunk 1]
```diff
@@ -101,13 +137,19 @@
         var htmlItem = $template.clone().removeClass('template');
         $('.link', htmlItem).html(item.label || item.__str__).data('item', item);
         that.$suggestList.append(htmlItem);
+        htmlItem.on('mouseenter', function() {
```
Comment (ar4s) [inline src/ralph/static/src/js/ralph-autocomplete.js]: rm to 145 and add `hover` in css

File: src/ralph/static/src/scss/components/autocomplete_field.scss [hunk 2]
```diff
@@ -125,11 +125,11 @@ $font-size: rem-calc(16);
             a {
                 display: block;
                 padding: 5px;
-                &:hover {
-                    background-color: #fafafa;
-                }
             }
         }
+        .selected {
```
Comment (ar4s) [inline src/ralph/static/src/scss/components/autocomplete_field.scss]: `.selected, &:hover`

**Comment by mkurek** [comment]
fix @ar4s's notices and it's :ok: for me

**Comment by vi4m** [comment]
I love it! This small things make Ralph great!

**Comment by ar4s** [comment]
:ok:

**Pull request merged**